src/callbacks/onPreprocess/definePopulationDetails.js

```diff
@@ -0,0 +1,54 @@
+import { set, time } from 'd3';
+import updatePopulationDetails from './definePopulationDetails/updatePopulationDetails';
+
```

Comment:
Assuming this is where the bug in #108 lives. Probably easiest just to fix it in this PR, yes? I think we're going to have to deal with minor merge conflicts anyway, so probably no harm done ... your call.